### ISSUE TITLE
certain blocks end up with combined line numbers instead of separate

### ISSUE THREAD

**Issue by kapilt** [cloud-custodian/tfparse#26]
locals and provider for example

```
>>> pprint.pprint(x['locals'])
{'__tfmeta': [{'filename': 'db.tf', 'line_end': 6, 'line_start': 1},
              {'filename': 'ecs_task.tf', 'line_end': 29, 'line_start': 2},
              {'filename': 'locals.tf', 'line_end': 69, 'line_start': 23}],
```

**Comment by sontek** [comment]
@kapilt Do you have a terraform file that can be used to replicate this issue?

**Comment by wwitzel3** [comment]
closing, I believe this is fixed by #37 
please re-open with example TF if you find it to still be producing unexpected results
